src/plugins/workspace/server/utils.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import crypto from 'crypto';
+
+/**
+ * Generate URL friendly random ID
+ */
+export const generateRandomId = (size: number) => {
+  return crypto.randomBytes(size).toString('base64url').slice(0, size);
```

Comment:
According this [RFC](https://datatracker.ietf.org/doc/html/rfc4648#section-5), the `base64url` will includes symbol like "_" and "-". Is this by  intention?

Reply:
yes, it should be url safe, any concerns?